jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/internal/compress/ByteAccumulator.java

```diff
@@ -22,56 +22,77 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import org.eclipse.jetty.io.ByteBufferPool;
+import org.eclipse.jetty.util.BufferUtil;
 import org.eclipse.jetty.websocket.core.MessageTooLargeException;
 
 /**
- * Collect up 1 or more byte arrays for later transfer to a single {@link ByteBuffer}.
+ * Collect up 1 or more ByteBuffers for later transfer to a single {@link ByteBuffer}.
  * <p>
  * Used by decompression routines to fail if there is excessive inflation of the
  * decompressed data. (either maliciously or accidentally)
  * </p>
  */
 public class ByteAccumulator
```

Comment:
What needs this class now?

Reply:
Only used by `compress()` and `decompress()` in `CompressExtension`.